cleanlab/classification.py

```diff
@@ -100,14 +101,14 @@ def score(self, X, y, sample_weight = None):
     compute_py_inv_noise_matrix,
     compute_noise_matrix_from_inverse,
 )
-from cleanlab.pruning import get_noise_indices
+from cleanlab.filter import find_label_issues
 
 
 class LearningWithNoisyLabels(BaseEstimator):  # Inherits sklearn classifier
     """Automated learning with noisy labels using any model.
 
     Confident Learning is the state-of-the-art (Northcutt et al., 2021) for
```

Comment:
This docstring seems like an odd place to define confident learning and talk about its miscellaneous capabilities for eg. finding label issues, uncertainty estimation, etc.  Shouldn't the docstring describe what this class does instead? Ie. it produces a new version of the same model as `clf` which has been trained in a way that is more robust to label noise in the data.

Reply:
done